apps/docs/src/pages/components/code-editor.mdx

```diff
@@ -0,0 +1,243 @@
+import { Callout } from "nextra/components";
+import {
+  codeEditorClasses,
+  HvCodeEditor,
+} from "@hitachivantara/uikit-react-code-editor";
+
+import Playground from "@docs/components/code/Playground";
+import { Description } from "@docs/components/page/Description";
+import { Page } from "@docs/components/page/Page";
+import { getComponentData } from "@docs/utils/component";
+
+export const getStaticProps = async ({ params }) => {
+  const meta = await getComponentData(
+    "CodeEditor",
+    "code-editor",
+    codeEditorClasses,
+  );
+  return { props: { ssg: { meta } } };
+};
+
+<Description />
+
+<Page>
+
+### Usage
+
+To use the Code Editor component you'll have to install it from the `@hitachivantara/uikit-react-code-editor` package:
+
+```
+npm install @hitachivantara/uikit-react-code-editor
+```
+
+To use it in your app, import it from the `"@hitachivantara/uikit-react-code-editor` package and it's ready to use:
```

Comment:
duplicate? we're saying `@hitachivantara/uikit-react-code-editor` 3 times in 3 lines. the codeblock should suffice

Reply:
I'll improve this.